hazelcast-jet-hadoop/src/main/java/com/hazelcast/jet/connector/hadoop/WritableSerializerHook.java

```diff
@@ -25,56 +24,58 @@
 import org.apache.hadoop.io.Writable;
 
 import java.io.IOException;
+import java.util.function.Supplier;
 
-import static com.hazelcast.jet.impl.util.ExceptionUtil.rethrow;
-
-public class WritableSerializerHook {
+/**
+ * This class can be extended to quickly create strongly typed serializers for
+ * {@code Writable} types
```

Comment:
I couldn't understand this doc. The class seems to support quick operation without the need to extend it and the actual subclasses provided don't extend anything. Is the requirement of the serializer hook mechanism to have distinct hook types?